posthog/test/test_cloud_utils.py

```diff
@@ -9,7 +9,7 @@
 
 class TestCloudUtils(BaseTest):
     def setUp(self):
-        assert len(License.objects.all()) == 0
+        assert License.objects.count() == 0
```

Comment:
Similar comment here, instead of:

* asking the DB to return all the records back to Python (burning CPU, network, memory allocations, etc...) to then count the number of objects

let's simply

* ask the DB to return the record count